README.md

```diff
@@ -1,8 +1,66 @@
-# Jarivs Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Sahar Soheilian Esfahani . Jarvis Consulting
 
+Completed an MSc in computer engineering at Shahid Beheshti University in Iran. Performance-driven and result-oriented developer with more than 5 years of experience in database design and project management in automation projects (BPMS). Experienced in leading engineering teams to achieve concrete goals on a strict deadline. Worked as a backend developer and implemeneted RESTful APIs in Django to support the platform's frontend and pre-process data. Skilled in Java, SQL, Python, Django, and docker. Seeking to leverage my expertise and knowledge to work on challenging problems and enhance product capabilities.
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git, BPMS, Docker, REST APIs, Postman
+
+**Competent:** Python, Django, Azure/GCP, C++, Microsoft Access, Microsoft Visio, Qt, Matlab
+
+**Familiar:** React, HTML, CSS, JavaScript, Microsoft Windows Server
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_SaharSoheilian](https://github.com/jarviscanada/jarvis_data_eng_SaharSoheilian)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_SaharSoheilian/tree/master/linux_sql)]: Had a Linux cluster of nodes/servers running CentOS 7 and internally connected through a switch. The project aimed to provide a monitoring agent which records hardware specifications of cluster nodes and monitors their resource usage in real-time. Used by Linux Cluster Administration (LCA) team to have a better insight into servers, and their usage to meet the business requirements. Technologies deployed in the project: Bash scripting, SQL, Git, GCP, and Docker.
```

Comment:
combine first two sentences
~~Had a Linux cluster of nodes/servers running CentOS 7 and internally connected through a switch~~ 

Developed a bash monitoring agent which records blah..blah in a Linux cluster

Reply:
The application is utilized by the LAC team ...

Reply:
~Technologies deployed in the project: Bash scripting, SQL, Git, GCP, and Docker.~

the application is implemented in Bash scripts, SQL, and PostgreSQL. Used docker and crontab to deploy the application. Github was used for source code management.